server/src/main/java/io/crate/analyze/AlterTableDropColumnAnalyzer.java

```diff
@@ -98,7 +96,7 @@ public AnalyzedAlterTableDropColumn analyze(AlterTableDropColumn<Expression> alt
 
     /** Validate restrictions based on properties that cannot change */
     private static void validateStatic(DocTableInfo tableInfo, List<DropColumn> dropColumns) {
-        if (tableInfo.versionCreated().before(Version.V_5_5_0)) {
+        if (tableInfo.versionCreated().before(DocTableInfo.COLUMN_OID_VERSION)) {
             throw new UnsupportedOperationException(
```

Comment:
We need the same for `AlterTableRenameColumnAnalyzer` and in `TransportAddColumnAction`. 
We have a check, I mean just using a new constant